wetterdienst/core/scalar/request.py

```diff
@@ -299,6 +299,18 @@ def __init__(
 
         self.tidy = tidy
 
+        log.info(
```

Comment:
We may as well move this to the .query() method and use the __str__/__repr__ representation of the class.

Reply:
Do you mean `wetterdienst.core.scalar.values.ScalarValuesCore.__str__`? Well, while I might be not be fully understanding it, currently it looks like this would have no way to see the whole picture like the `provider`.

I clearly believe that this logging function is at the right place here because it covers both requests for stations **and** values, that is what I was aiming at.